metadata/metadata.go

```diff
@@ -28,7 +28,8 @@ import (
 	"golang.org/x/net/context"
 )
 
-// DecodeKeyValue returns k, v, nil.  It is deprecated and should not be used.
+// DecodeKeyValue returns k, v, nil.
+// Deprecated; should not be used.
```

Comment:
Please use the "standard" deprecated syntax, so static analysis tools will notice it correctly:

```
// DecodeKeyValue returns k, v, nil.
//
// Deprecated: use k and v directly instead.
```